src/routes/common/logging/logging.module.ts

```diff
@@ -1,6 +1,31 @@
 import { Global, Module } from '@nestjs/common';
 import { LoggingService } from './logging.interface';
 import { RequestScopedLoggingService } from './logging.service';
+import * as winston from 'winston';
+import { format } from 'winston';
```

Comment:
Yeah good point. Since we already have the name `winston` given to all the symbols available under `winston` I decided to call it via `winston.format`: [0521370](https://github.com/safe-global/safe-gelato-relay-service/pull/62/commits/0521370dc6df1748588dbd2a4b6357aac1cfe920)